core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/number/NumberFormat.java

```diff
@@ -0,0 +1,36 @@
+package io.metadew.iesi.script.execution.instruction.data.number;
+
+import io.metadew.iesi.script.execution.instruction.data.DataInstruction;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class NumberFormat implements DataInstruction {
+
+    private final static String NUMBER = "number";
+    private final static String FORMAT = "format";
+
+    private final static Pattern PATTERN = Pattern.compile("\\s*\"(?<" + NUMBER + ">.+)\"\\s*,\\s*\"(?<" + FORMAT + ">.+)\"");
```

Comment:
can you make the double quotes surrounding the NUMBER optional

Reply:
Done